src/plugins/intel_cpu/src/nodes/executors/acl/acl_deconv.hpp

```diff
@@ -0,0 +1,183 @@
+        }
+
+        arm_compute::TensorInfo srcTensorInfo = arm_compute::TensorInfo(shapeCast(srcDims), 1,
+                                              precisionToAclDataType(srcDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(srcDescs[0]));
+        arm_compute::TensorInfo weiTensorInfo = arm_compute::TensorInfo(shapeCast(weiDims), 1,
+                                              precisionToAclDataType(srcDescs[1]->getPrecision()), getAclDataLayoutByMemoryDesc(srcDescs[1]));
+        arm_compute::TensorInfo dstTensorInfo = arm_compute::TensorInfo(shapeCast(dstDims), 1,
+                                              precisionToAclDataType(dstDescs[0]->getPrecision()), getAclDataLayoutByMemoryDesc(dstDescs[0]));
+
+        unsigned int pad_l = deconvAttrs.paddingL.at(1);
+        unsigned int pad_r = deconvAttrs.paddingR.at(1);
+        unsigned int pad_t = deconvAttrs.paddingL.at(0);
+        unsigned int pad_b = deconvAttrs.paddingR.at(0);
+        unsigned int stride_x = deconvAttrs.stride.at(1);
+        unsigned int stride_y = deconvAttrs.stride.at(0);
+
+        arm_compute::PadStrideInfo deconv_info(stride_x, stride_y, pad_l, pad_r, pad_t, pad_b, arm_compute::DimensionRoundingType::FLOOR);
+
+        size_t in_h = srcDescs[0]->hasLayoutType(LayoutType::ncsp) ? srcDims[2] : srcDims[1];
+        size_t in_w = srcDescs[0]->hasLayoutType(LayoutType::ncsp) ? srcDims[3] : srcDims[2];
+        if (validate_deconvolution_output_dimensions(in_w, in_h, deconvAttrs.kernel.at(1), deconvAttrs.kernel.at(0), deconv_info)) {
+            DEBUG_LOG("NEDeconvolutionLayer arm_compute::deconvolution_output_dimensions failed");
+            return false;
+        }
+
+        arm_compute::Status status = arm_compute::NEDeconvolutionLayer::validate(&srcTensorInfo,
+                                                                                 &weiTensorInfo,
+                                                                                 deconvAttrs.withBiases ? &biasTensorInfo : nullptr,
+                                                                                 &dstTensorInfo,
+                                                                                 deconv_info);
+        if (!status) {
+            DEBUG_LOG("NEDeconvolutionLayer validation failed: ", status.error_description());
+            return false;
+        }
+
+        return true;
+    }
+
+    bool isSupported(const DeconvAttrs& deconvAttrs,
+                     const std::vector<MemoryDescPtr>& srcDescs,
+                     const std::vector<MemoryDescPtr>& dstDescs) const override {
+        return customIsSupported(deconvAttrs, srcDescs, dstDescs);
+    }
+
+    DeconvExecutorPtr makeExecutor(const ExecutorContext::CPtr context) const override {
+        return std::make_shared<AclDeconvExecutor>(context);
+    }
+
+private:
+    static bool validate_deconvolution_output_dimensions(unsigned int in_width, unsigned int in_height,
```

Comment:
looks like this method is copied from ACL.
Shouldn't we rely on NEDeconvolutionLayer::validate? I assume it makes the same checks as well.

Reply:
@alvoron done